sklearn/svm/base.py

```diff
@@ -779,6 +779,12 @@ def _fit_liblinear(X, y, C, fit_intercept, intercept_scaling, class_weight,
     if verbose:
         print('[LibLinear]', end='')
 
+    # LinearSVC breaks when intercept_scaling is 0
+    if intercept_scaling is 0:
```

Comment:
@amueller - you mean that I should add a `fit_intercept=True` condition here and in the tests too?